caluma/caluma_form/tests/__snapshots__/test_question.ambr

```diff
@@ -881,6 +883,27 @@
     }),
   })
 # ---
+# name: test_save_float_question[float-question__configuration2-0.3-True]
+  dict({
+    'saveFloatQuestion': dict({
+      'clientMutationId': 'testid',
+      'question': dict({
+        '__typename': 'FloatQuestion',
+        'defaultAnswer': dict({
+          'value': 0.3,
+        }),
+        'hintText': 'test',
+        'id': 'RmxvYXRRdWVzdGlvbjplbnZpcm9ubWVudGFsLXRlbg==',
+        'label': 'Bonnie Moreno',
+        'maxValue': None,
+        'meta': dict({
+        }),
+        'minValue': None,
```

Comment:
shouldn't the step value be in here as well?

Reply:
No the value is validated to be `>0` and the `step` value here is `-0.3`. Second param is `step`:
> `# name: test_save_float_question[float-question__configuration2-0.3-True]`